VotingApplication/VotingApplication.Web/Scripts/Controllers/BasicVoteController.js

```diff
@@ -1,54 +1,55 @@
 ﻿(function () {
-    angular.module('GVA.Voting').controller('BasicVoteController', ['$scope', 'IdentityService', 'PollService', 'TokenService', function ($scope, IdentityService, PollService, TokenService) {
-
-        var pollId = PollService.currentPollId();
-        var token = null;
-
-        $scope.vote = function (option) {
-            if (!option) {
-                return null;
-            }
-
-            if (!token) {
-                // Probably invite only, tell the user
-            } else if (!IdentityService.identity) {
-                IdentityService.openLoginDialog($scope, function () {
-                    $scope.vote(option);
-                });
-            } else {
-                votes = [{
-                    OptionId: option.Id,
-                    VoteValue: 1,
-                    VoterName: IdentityService.identity.name
-                }];
-
-                PollService.submitVote(pollId, votes, token, function () {
-                    window.location = $scope.$parent.resultsLink;
-                });
+    angular.module('GVA.Voting').controller('BasicVoteController', ['$scope', '$routeParams', 'IdentityService', 'PollService', 'TokenService',
```

Comment:
This is rather a long line. Would be a bit easier on the eyes if .module and .controller were each on their own line.